docs/pages/guides/building-for-tv.mdx

```diff
@@ -0,0 +1,222 @@
+- **Android TV**:
+  - In the Android Studio SDK manager, select the dropdown for the Android SDK you are using (API version 31 or later), and make sure an Android TV system image is selected for installation. (For Apple Silicon Macs, choose the ARM 64 image, otherwise, choose the Intel x86_64 image).
+  - After installing the Android TV system image, create an Android TV emulator using that image (the process is the same as for creating an Android phone emulator).
+
+> **Info** If you are starting with an SDK 49 project, you will need to [upgrade to SDK 50 or higher](/workflow/upgrading-expo-sdk-walkthrough/) before proceeding through the following steps.
+
+## Limitations
+
+Certain Expo tools are not yet supported at this time, most notably:
+
+- expo-dev-client
+- Expo Router
+
+<Step label="1">
+
+## Modify dependencies for TV
+
+In **package.json**, modify the `react-native` dependency to use the TV repo, and exclude this dependency from [`npx expo install` version validation](/more/expo-cli/#configuring-dependency-validation).
+
+{/* prettier-ignore */}
+```json package.json
+{
+  /* @hide ... your existing package.json */ /* @end */
+  "dependencies": {
+    /* @hide ... your existing dependencies */ /* @end */
+    "react-native": "npm:react-native-tvos",
+    /* @hide ... your existing dependencies */ /* @end */
+  },
+  "expo": {
+    "install": {
+      "exclude": [
+        "react-native"
+      ]
+    }
+  }
+}
+```
+</Step>
+
+<Step label="2">
+
+## Add the TV config plugin
+
+<Terminal cmd={['npx expo install @react-native-tvos/config-tv']} />
+
+Installing the package adds the `@react-native-tvos/config-tv` plugin to your project's **app.json**.
+
+When installed, the plugin will modify the project for TV when either:
+
+- The environment variable `EXPO_TV` is set to `1`
+- The plugin parameter `isTV` is set to `true`
+
+Optionally, set the `showVerboseWarnings` property to `true` to see additional information during prebuild.
+
+```json app.json
+{
+  "plugins": [
+    [
+      "@react-native-tvos/config-tv",
+      {
+        "showVerboseWarnings": true
+      }
+    ]
+  ]
+}
+```
+
+</Step>
+
+<Step label="3">
+
+## Run prebuild
+
+Set the `EXPO_TV` environment variable, and run prebuild to make the TV modifications to the project.
+
+<Terminal cmd={['export EXPO_TV=1', 'npx expo prebuild --clean']} />
+
+> **Note**: The `--clean` argument is recommended, and is required if you have existing Android and iOS directories in the project.
+
+</Step>
+
+<Step label="4">
+
+## Build for Apple TV
+
+After running prebuild, run the following command to build and run the app on an Apple TV simulator:
+
+<Terminal
+  cmd={[
+    'npx expo run:ios'
+  ]}
+/>
+
+</Step>
+
+<Step label="5">
+
+## Build for Android TV
+
+After running prebuild, start an Android TV emulator, and then use the same command as you would to start the app on an Android phone emulator.
```

Comment:
```suggestion
Start an Android TV emulator and use the following command to start the app on the emulator:
```

In some cases, people might not be familiar with `npx expo run...` commands. So let's not reference that.